src/components/map/layer-hover-info.js

```diff
@@ -81,7 +113,27 @@ const EntryInfoRow = ({item, fields, data}) => {
   const displayValue = item.format
     ? getFormatter(item.format, field)(data[valueIdx])
     : parseFieldValue(data[valueIdx], field.type);
-  return <Row name={item.name} value={displayValue} />;
+  let displayDeltaValue = null;
+  if (
+    primaryData &&
+    (field.type === ALL_FIELD_TYPES.integer || field.type === ALL_FIELD_TYPES.real)
+  ) {
+    const deltaValue =
+      compareType === COMPARE_TYPES.PERCENTAGE
+        ? 1 - data[valueIdx] / primaryData[valueIdx]
+        : data[valueIdx] - primaryData[valueIdx];
+    const deltaFormat =
+      compareType === COMPARE_TYPES.PERCENTAGE
+        ? TOOLTIP_FORMATS.DECIMAL_PERCENT_FULL_2[TOOLTIP_KEY]
+        : TOOLTIP_FORMATS.DECIMAL_DECIMAL_FIXED_3[TOOLTIP_KEY];
+    displayDeltaValue = getFormatter(item.format || deltaFormat)(deltaValue);
+    const deltaFirstChar = displayDeltaValue.toString().charAt(0);
```

Comment:
why not add another format type: `TOOLTIP_FORMATS.DELTA` and use d3.format('+')(value)

Reply:
If we have % in the string it would be NaN